### DISCUSSION TITLE
[Config Support]: Scaling and framerate

### DISCUSSION THREAD

**Discussion by ProsjektX** [blakeblackshear/frigate#14624] [Config Support]
### Describe the problem you are having

If one wants to use a high-resolution and higher framerate stream as basis for detect, what are the differences and advantages, if any, to either (assuming hw acceleration defined in the config file)

1. pass the stream through go2rtc and let ffmpeg do downscaling and fps reduction
2. define the width, height and fps under the camera section for detect

Sample go2rtc stream below as reference
- ffmpeg:rtsp://user:pass@192.168.1.108:554/cam/realmonitor?channel=1&subtype=0#hardware#video=h264#width=1280#height=720#raw=-r 5

### Version

0.14.1

### Frigate config file

```yaml
<not relevant>
```

### Relevant Frigate log output

```shell
<not relevant>
```

### Relevant go2rtc log output

```shell
<not relevant>
```

### Operating system

UNRAID

### Install method

Docker Compose

### docker-compose file or Docker CLI command

```yaml
<not relevant>
```

### Object Detector

OpenVino

**Comment by NickM-27** [comment]
If you're not using the stream elsewhere there is no reason to have go2rtc do this.
